OutOfSchool/OutOfSchool.WebApi/Services/MinistryAdminService.cs

```diff
@@ -0,0 +1,235 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+using Newtonsoft.Json;
+
+using OutOfSchool.Common;
+using OutOfSchool.Common.Models;
+using OutOfSchool.Services.Models;
+using OutOfSchool.Services.Repository;
+using OutOfSchool.WebApi.Config;
+using OutOfSchool.WebApi.Enums;
+using OutOfSchool.WebApi.Extensions;
+using OutOfSchool.WebApi.Models;
+using OutOfSchool.WebApi.Services.Communication;
+
+namespace OutOfSchool.WebApi.Services
+{
+    public class MinistryAdminService : CommunicationService, IMinistryAdminService
+    {
+        private readonly IdentityServerConfig identityServerConfig;
+        private readonly IEntityRepository<User> userRepository;
+        private readonly IMinistryAdminRepository ministryAdminRepository;
+        private readonly ILogger<MinistryAdminService> logger;
+        private readonly IMapper mapper;
+        private readonly ResponseDto responseDto;
```

Comment:
What's the purpose to have `responseDto` as sevice member? I'm wondering because if somebody somehow calls this service from differrent threads, there will be a problem that we can receive unexpected results without even any exception.
Or another case is that 2+ subsequential calls will change one model so this is delayed problem until somebody forget that you change only 2 fields from that model
```
responseDto.IsSuccess = false;
responseDto.HttpStatusCode = HttpStatusCode.NotFound;
```